packages/main/src/plugin/mcp/mcp-schema-validator.spec.ts

```
@@ -0,0 +1,266 @@
+              description: 'test',
+              version: '1.0.0',
+            },
+            // Missing required '_meta' field
+          },
+        ],
+      };
+
+      validator.validateSchemaData(invalidData, 'ServerList', 'https://example-registry.com');
+
+      expect(console.warn).toHaveBeenCalledWith(
+        expect.stringContaining(`from https://example-registry.com`),
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)